src/autopas/cells/FullSortedParticleCell.h

```diff
@@ -0,0 +1,81 @@
+/**
+ * @file FullSortedParticleCell.h
```

Comment:
Wouldn't `SortedFullParticleCell` fit better? The cells are not fully sorted.

Reply:
I agree, I'll rename the class to `SortedCellView`